content/includes/nginx-one/cloud-access.md

```diff
@@ -4,15 +4,22 @@ files:
   - content/nginx-one/getting-started.md
```

Comment:
If it's not a hassle, can you also add this file to the list:

`content/nginx-one/workshops/lab1/getting-started-with-nginx-one.md`

I used your include in the Lab 1 workshop. :)

Reply:
```suggestion
  - content/nginx-one/getting-started.md
  - content/nginx-one/workshops/lab1/getting-started-with-nginx-one.md
```